Annotato/Annotato/ViewModels/Document/DocumentViewModel.swift

```diff
@@ -53,4 +52,9 @@ extension DocumentViewModel {
         annotations.append(annotationViewModel)
         addedAnnotation = annotationViewModel
     }
+
+    func removeAnnotation(annotation: AnnotationViewModel) {
+        model.removeAnnotation(annotation: annotation.model)
+        annotations.removeAll(where: { $0 === annotation })
```

Comment:
Should we just use normal equality here with ID? I think that's how gn does it in the rest of the codebase

Reply:
Yea I think that's safer. Was wondering if we should have made classes conform to Equatable too but I didn't cos it seemed like too much work lol.